example/ios/Podfile.lock

```diff
@@ -1586,8 +1586,8 @@ SPEC CHECKSUMS:
   RNGestureHandler: a479ebd5ed4221a810967000735517df0d2db211
   RNScreens: 23dad53fc9db1da2c93e647ae33fd7ce2bd49d60
   SocketRocket: abac6f5de4d4d62d24e11868d7a2f427e0ef940d
-  Yoga: 04f1db30bb810187397fa4c37dd1868a27af229c
+  Yoga: 88480008ccacea6301ff7bf58726e27a72931c8d
 
 PODFILE CHECKSUM: 54f3e178906b491e7b20bd99da6e8788777ec12e
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.13.0
```

Comment:
probably you need to update your cocoapods instance and run it again, otherwise I don't see why we need to downgrade version used to generate this `lock` file

Reply:
totally not a blocker

Reply:
We fixed this still 💯